codecov.yml

```diff
@@ -1,12 +1,17 @@
 codecov:
-  token: 85c13d8a-e9e1-49fd-97d4-7ba103f3e830
```

Comment:
The token is now set as a GitHub secret called `CODECOV_TOKEN`, so it's redundant to have it here too.  It seemed like this setting has been ignored.  There's a chance this change may break something though.

Reply:
Might be worth a breaking.rst incase people have used to token to set up there own repos on their local machines

Reply:
I believe the only usage of this token has been via the `main` branch of the original GitHub repo.